### ISSUE TITLE
Undefine type

### ISSUE THREAD

**Issue by thevan96** [marlonfan/coc-phpls#47]
**Describe the bug**
Show error Undefined type 'Route'

**To Reproduce**
1. Create project laravel : laravel new blog
2. Open file routes/web.php
3. Show Error

**Expected behavior**
Auto show error

**Screenshots**
![image](https://user-images.githubusercontent.com/47932916/71794666-20557600-3075-11ea-8c95-225a52c67d74.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version : Laravel 6.9

**Additional context**
Use coc-phpls in neovim

**Comment by marlonfan** [comment]
![image](https://user-images.githubusercontent.com/9349370/71871903-beb90880-3155-11ea-80d4-79fc54caf731.png)
``Rout`` -> ``Route``

**Comment by thevan96** [comment]
sorry i write wrong, but Route return error
![image](https://user-images.githubusercontent.com/47932916/71907687-1af44a80-319f-11ea-8c05-fcf8271d13fa.png)

**Comment by marlonfan** [comment]
@thevan96 i think maybe you need laravel ide helper. [FYI](https://github.com/barryvdh/laravel-ide-helper)

**Comment by thevan96** [comment]
Haizzzz

**Comment by wilbura009** [comment]
tldr; run 

```
composer update
```

---

I just had this same issue. The previous day this plugin was working fine without the additional [laravel ide helper](https://github.com/barryvdh/laravel-ide-helper). I searched and found this solution on [StackOverFlow](https://stackoverflow.com/questions/26887111/laravel-php-fatal-error-require-failed-opening-required-bootstrap-autoload). I ran `composer update` and it got rid of the errors for me.
